examples/event-sourced-domain-with-tests/InvitesTest.php

```diff
@@ -23,7 +23,7 @@ public function setUp()
         $this->generator = new Broadway\UuidGenerator\Rfc4122\Version4Generator();
     }
 
-    protected function getAggregateRootClass()
+    protected function getAggregateRootClass():string
```

Comment:
protected function getAggregateRootClass(): string

sorry, the CS fixer is picky